src/widget/wsearchlineedit.cpp

```diff
@@ -208,12 +208,15 @@ void WSearchLineEdit::showPlaceholder() {
     m_state = State::Inactive;
 
     setText(tr("Search...", "noun"));
-    setToolTip(
-            tr("Search", "noun") + "\n" + tr("Enter a string to search for") + "\n\n" +
-            tr("Shortcut") + ": \n" + tr("Ctrl+F") + "  " +
-            tr("Focus", "Give search bar input focus") + "\n" + tr("Ctrl+Backspace") + "  " +
-            tr("Clear input", "Clear the search bar input field") + "\n" + tr("Esc") + "  " +
-            tr("Exit search", "Exit search bar and leave focus"));
+    setToolTip(tr("Search", "noun") + "\n" +
+            tr("Enter a string to search for") + "\n" +
+            tr("Use operators like bpm:115-128, artist:BooFar, -year:1990") + "\n" +
+            tr("See chapter 4.4.1 in the Manual for more information") + "\n\n" +
```

Comment:
I wonder if it is reasonable to have here the exact chapter number. 
This might float, without knowing that It is referenced here. On the other hand it is the easiers reference and translations proof. Can we put a comment about that relationship to the user manual source?  

Thoughts?

Reply:
right, the chapter might be moved.
I don't know how most people work on the manual, but I think a comment for reference might slip through when the chapter is moved. A fixed reference is better as the chapter title is less likely to change essentially.
I just thought the literal name might be a very long string in some languages..
`See User Manual > Mixxx Library > Finding Tracks for more information`
`See chapter 4.4.1 in the Manual for more information`

what about just
`For more information see User Manual > Mixxx Library`
?

Reply:
Yes, that works for me. If we use the actual headline, we have the issue that the translations might be different, because it may be done by different people.